### PR TITLE
 EASY-2040: Make springfield-actions namespace a URI

### PR DESCRIPTION
**Pull request by lindareijnhoudt**
accompanies EASY-2040

#### How should this be manually tested?
there are no example xml's in easy-schema-examples for this xsd

### CODE REVIEW

File: src/main/assembly/dist/extern/springfield-actions.xsd [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xs:schema xmlns:sf="springfield-actions" xmlns:xml="http://www.w3.org/XML/1998/namespace"
-           targetNamespace="springfield-actions" xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified">
+<xs:schema xmlns:sf="http://easy.dans.knaw.nl/extern/springfield-actions/" xmlns:xml="http://www.w3.org/XML/1998/namespace"
```
Comment (janvanmansum) [inline src/main/assembly/dist/extern/springfield-actions.xsd]: Shouldn't that be "external"?

Comment (lindareijnhoudt) [reply src/main/assembly/dist/extern/springfield-actions.xsd]: maybe that's better. i just copied the word from the `easy-schema` URL

Comment (rvanheest) [reply src/main/assembly/dist/extern/springfield-actions.xsd]: in that case, I probably merged https://github.com/DANS-KNAW/easy-ingest-flow/pull/108 too quickly 🤔

Comment (stephanHogenboom) [reply src/main/assembly/dist/extern/springfield-actions.xsd]: Okay let us know if it becomes external, as we need to change it in easy-ingest-flow as well, as Richard indicated.

Comment (lindareijnhoudt) [reply src/main/assembly/dist/extern/springfield-actions.xsd]: @rvanheest : i've changed it from 'extern' to 'external'

Comment (rvanheest) [reply src/main/assembly/dist/extern/springfield-actions.xsd]: I've also changed it in https://github.com/DANS-KNAW/easy-ingest-flow/pull/111

**Pull request merged**